v8-compile-cache.js

```diff
@@ -100,7 +100,7 @@ class FileSystemBlobStore {
       this._storedBlob = fs.readFileSync(this._blobFilename);
       this._storedMap = JSON.parse(fs.readFileSync(this._mapFilename));
     } catch (e) {
-      this._storedBlob = new Buffer(0);
+      this._storedBlob = Buffer.alloc(0);
```

Comment:
This is just a dummy fallback buffer. Can you use `Buffer.from([])` so that this passed in 5.7.0 please?